tests/ci/worker/init_runner.sh

```diff
@@ -98,24 +180,50 @@ if [ "$(docker ps --all --quiet)" ]; then
     docker info && break || sleep 2
   done
   # Last chance, otherwise we have to terminate poor instance
-  docker info 1>/dev/null || { echo Docker unable to start; terminate-and-exit; }
+  docker info 1>/dev/null || { echo Docker unable to start; terminate_delayed ; }
 fi
 EOF
 
+get_runner_token() {
+    /usr/local/bin/aws ssm  get-parameter --name github_runner_registration_token --with-decryption --output text --query Parameter.Value
+}
+
+is_job_assigned() {
+    local runner_pid
+    runner_pid=$(pgrep Runner.Listener)
+    if [ -z "$runner_pid" ]; then
+        # if runner has finished, it's fine
+        return 0
+    fi
+    local log_file
+    log_file=$(lsof -p "$runner_pid" 2>/dev/null | grep -o "$RUNNER_HOME/_diag/Runner.*log")
```

Comment:
This may be incorrect, I do see log files in different format `Runner_20230607-142302-utc.log`
```suggestion
    log_file=$(lsof -p "$runner_pid" 2>/dev/null | grep -o "$RUNNER_HOME/_diag/Runner*.log")
```